maps/snaxi.dmm

```diff
@@ -518,7 +518,7 @@
 "ajX" = (/obj/structure/table,/obj/item/weapon/storage/firstaid/fire{pixel_x = 2; pixel_y = 6},/obj/item/weapon/storage/firstaid/regular{pixel_x = -2; pixel_y = 4},/obj/effect/landmark{name = "xeno_spawn"; pixel_x = -1},/turf/simulated/floor{icon_state = "floorgrime"},/area/storage/primary)
 "ajY" = (/turf/simulated/wall,/area/storage/primary)
 "ajZ" = (/turf/simulated/floor{icon_state = "freezerfloor"},/area/maintenance/fsmaint)
-"aka" = (/turf/simulated/wall/invulnerable/ice,/area)
+"aka" = (/obj/machinery/door/airlock/external{req_access_txt = "13"},/obj/structure/window/barricade/full/block,/turf/simulated/floor/plating,/area/hallway/primary/aft{name = "\improper Central Civillian Hallway"})
```

Comment:
Yo I thought I fixed it being called "Civillian" but here it is.

Reply:
If you make an issue over it, I'll do that in my bigger PR since it's mostly cosmetic.

Reply:
No no it's not an issue worth addressing here I'm just perplexed because I thought I edited the area name and it should have all been updated.  I messed up.